app/src/organisms/GripperCard/__tests__/GripperCard.test.tsx

```diff
@@ -0,0 +1,131 @@
+      } as GripperData,
+      isCalibrated: true,
+    }
+    mockBanner.mockReturnValue(<>calibration needed</>)
+    mockGripperWizardFlows.mockReturnValue(<>wizard flow launched</>)
+    mockAboutGripperSlideout.mockReturnValue(<>about gripper</>)
+  })
+  afterEach(() => {
+    jest.resetAllMocks()
+    resetAllWhenMocks()
+  })
+
+  it('renders correct info when gripper is attached', () => {
+    const { getByText, getByRole } = render(props)
+    const image = getByRole('img', { name: 'flex gripper' })
+    expect(image.getAttribute('src')).toEqual('flex_gripper.png')
+    getByText('extension mount')
+    getByText('Flex Gripper')
+    const overflowButton = getByRole('button', {
+      name: /overflow/i,
+    })
+    fireEvent.click(overflowButton)
+    getByText('Recalibrate gripper')
+    getByText('Detach gripper')
+    getByText('About gripper')
+  })
+  it('renders recalibrate banner when no calibration data is present', () => {
+    props = props = {
+      attachedGripper: {
+        instrumentModel: 'gripperV1.1',
+        serialNumber: '123',
+        firmwareVersion: '12',
+        data: {
+          calibratedOffset: {
+            last_modified: undefined,
+          },
+        },
+      } as GripperData,
+      isCalibrated: false,
+    }
+
+    const { getByText } = render(props)
+    getByText('calibration needed')
+  })
+  it('opens the about gripper slideout when button is pressed', () => {
+    const { getByText, getByRole } = render(props)
+    const overflowButton = getByRole('button', {
+      name: /overflow/i,
+    })
+    fireEvent.click(overflowButton)
+    const aboutGripperButton = getByText('About gripper')
+    fireEvent.click(aboutGripperButton)
+    getByText('about gripper')
+  })
+  it('renders wizard flow when recalibrate button is pressed', () => {
+    const { getByText, getByRole } = render(props)
+    const overflowButton = getByRole('button', {
+      name: /overflow/i,
+    })
+    fireEvent.click(overflowButton)
+    const recalibrateGripperButton = getByText('Recalibrate gripper')
+    fireEvent.click(recalibrateGripperButton)
+    getByText('wizard flow launched')
+  })
+  it('renders wizard flow when detach button is pressed', () => {
+    const { getByText, getByRole } = render(props)
+    const overflowButton = getByRole('button', {
+      name: /InstrumentCard_overflowMenu/i,
+    })
+    fireEvent.click(overflowButton)
+    const detachGripperButton = getByText('Detach gripper')
+    fireEvent.click(detachGripperButton)
+    getByText('wizard flow launched')
+  })
+  it('renders wizard flow when attach button is pressed', () => {
+    props = {
+      attachedGripper: null,
+      isCalibrated: false,
+    }
+    const { getByText, getByRole } = render(props)
+    const overflowButton = getByRole('button', {
+      name: /overflow/i,
+    })
+    fireEvent.click(overflowButton)
+    const attachGripperButton = getByText('Attach gripper')
+    fireEvent.click(attachGripperButton)
```

Comment:
nit
recently we use .click() instead of fireEvent
```suggestion
    attachGripperButton.click()
```